hyperspace/testsuite/src/lib.rs

```diff
@@ -594,3 +594,32 @@ where
 	send_packet_and_assert_timeout_on_channel_close(chain_a, chain_b, channel_id).await;
 	handle.abort()
 }
+
+pub async fn client_synchronization_test<A, B>(chain_a: &A, chain_b: &B)
```

Comment:
is it possible to have a test in which the client isn't synced, and then it becomes synced? Maybe this is exactly what this test does, and I'm not following it 100%

Reply:
Yeah that's what this test does, it sleeps for while after client creation, some sessions pass and the clients become out of sync.